src/bin.js

```diff
@@ -41,21 +40,17 @@ function parseOptions(args, availableOptions) {
       console.log("We suggest using the new 'overrides' option instead of nested .babelrc's, can check out http://new.babeljs.io/docs/en/next/babelrc.html#overrides");
       console.log("");
     }
-    paths.forEach(p => writeBabelRC(p, upgradeOptions));
+
+    for (let path of paths)  writeBabelRC(path, upgradeOptions);
```

Comment:
Any reason for this change?

Reply:
It will be more efficient to use keywords & statements than methods & callbacks, because Babel 7 core has dropped the support of old Node.JS.

Reply:
This isn’t actually more efficient - it uses the iterator protocol instead of the quote fast array method.

Older stuff is almost always faster than newer stuff.

Reply:
Not in latest v8 - they made the array methods identically as fast as for loops.